### PR TITLE
Document existing aspects of the Env feature(beta)

### PR DESCRIPTION
**Pull request by HalynaDumych**
Add the Environment section with the following instructions:

* Find environments to which you have access
* Change your active environment
* Add members to environments
* Remove members from environments

Relates-to: OTA-4488

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: ota-plus-web/docs/modules/ROOT/pages/add-members.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+= Add members to environments (Beta)
+
+[NOTE]
+====
+It is a beta feature, and only Beta Customers can access it. We will inform you when the feature is released officially.
```
Comment (tkfu) [inline ota-plus-web/docs/modules/ROOT/pages/add-members.adoc]:
```suggestion
This feature is in beta, and is only visible to users who are part of our beta program. If you would like to join the beta program, contact us at link:mailto:otaconnect.support@here.com[otaconnect.support@here.com] to request access.
```

Go with this text for the notices on all the pages, please.

File: ota-plus-web/docs/modules/ROOT/pages/add-members.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+= Add members to environments (Beta)
```
Comment (tkfu) [inline ota-plus-web/docs/modules/ROOT/pages/add-members.adoc]:
```suggestion
= Add members to environments
```

The beta tag will be on the whole category, no need to put it on every page. (Please also remove from the other page titles, and leave it only on the top-level title in the TOC.)

**Comment by tkfu** [comment]
@HalynaDumych, if you need some help adding screenshots, @ammaboateng can walk you through it. The quick version is this: you just add the images to the ota-connect-docs-images repo and reference them using antora's normal xref syntax.

**Comment by HalynaDumych** [comment]
> @HalynaDumych, if you need some help adding screenshots, @ammaboateng can walk you through it. The quick version is this: you just add the images to the ota-connect-docs-images repo and reference them using antora's normal xref syntax.

@tkfu, I didn't add the screenshots as the instructions are pretty obvious. If needed, I will add them. I don't think there might be some problems with it, but thank you for help!

**Comment by tkfu** [comment]
Yes, I suppose you're right...screenshots are probably overkill for this.

Thanks!

**Comment by HalynaDumych** [comment]
> Looks great to me--as far as I'm concerned, feel free to merge. But check in with @BeneBuz first, just in case she wants to coordinate it with customer communication or something.

Sure, thank you!

**Pull request merged**